types/nodecredstash/index.d.ts

```diff
@@ -0,0 +1,45 @@
+// Type definitions for nodecredstash 2.0
+// Project: https://github.com/DavidTanner/nodecredstash#readme
```

Comment:
Minor point, but the convention is to put the github link in here without the `#readme`